qtcodes/fitters/lattice_decoder.py

```diff
@@ -236,7 +235,7 @@ def _path_degeneracy_helper(
         elif b_indx in num_shortest_paths.keys():
             return num_shortest_paths[b_indx][a_indx], num_shortest_paths
         else:
-            num_shortest_paths[a_indx] = GraphUtils.num_shortest_paths(
+            num_shortest_paths[a_indx] = rx.num_shortest_paths_unweighted(
```

Comment:
Could we change this to: `dict(rx.num_shortest_paths_unweighted(...))` ?

Let's also adjust all the typing hints from `Dict[int, List[int]]` to `Dict[int, Dict[int, int]]`.

Reply:
Done in: https://github.com/yaleqc/qtcodes/pull/48/commits/c2080013e0a8810c98b0887819fa582c0b9c6edc